src/physics/nonlinear_solid.hpp

```diff
@@ -33,8 +34,9 @@ class NonlinearSolid : public BasePhysics {
    * @brief A timestep method and config for the M solver
    */
   struct DynamicSolverParameters {
-    TimestepMethod         timestepper;
-    LinearSolverParameters M_params;
+    TimestepMethod             timestepper;
+    DirichletEnforcementMethod enforcement_method;
+    LinearSolverParameters     M_params;
```

Comment:
This can be removed

Reply:
What do you mean?

Reply:
It doesn't look like the `M_params` is used anywhere - looks like GitHub displays the previous two lines but I just meant 39.

Reply:
ah, that makes more sense!